scripts/sdk/build.sh

```diff
@@ -10,17 +10,7 @@ mkdir target/paths
 cp doc/SDK.md target/README.md
 pushd target
 pushd asset-build/assets
-mv main.*.js ../../main.js
-# Create a copy of light theme for backwards compatibility 
-cp theme-element-light.*.css ../../style.css
-# Remove asset hash from css files
-mv theme-element-light.*.css ../../theme-element-light.css
-mv theme-element-dark.*.css ../../theme-element-dark.css
-mv download-sandbox.*.html ../../download-sandbox.html
-rm *.js *.wasm
-mv ./* ../../
+rm !(main).js *.wasm
```

Comment:
This needs `shopt -s extglob` to enable extended globs.

Seems to work fine with git bash on Windows by default but had issues on macOS

```
./scripts/sdk/build.sh: line 13: syntax error near unexpected token `('
./scripts/sdk/build.sh: line 13: `rm !(main).js *.wasm'
```